source/_components/stream.markdown

```diff
@@ -55,6 +55,11 @@ action:
     filename: '/tmp/my_stream.mp4'
 ```
 
+## {% linkable_title Streaming in Lovelace %}
+
+As of Homeassistant version 0.92 you can now live-stream a camera feed directly in lovelace.
+To do this add a [picture-entity](/lovelace/picture-entity/) card with your stream-ready camera entity to one of your lovelace views.
```

Comment:
Perhaps mention that it is possible with these 3: picture glance, picture entity and picture element

Reply:
Added the other two cards and explained the setup process a bit more thoroughly.